### ISSUE TITLE
Allow Custom Install Path within the Mobiflight Installer Tool !

### ISSUE THREAD

**Issue by pizman** [MobiFlight/MobiFlight-Connector#496]
**Is your feature request related to a problem? Please describe.**
MF Installer just "unpack" the MF Files in the Folder where we start the installer.exe  ( Same way as a self extracting EXE in the past.)
Lots of USers ( mostly at first days in MF) not know this.  And also advanced users make this mistake from time to time.
I will say 99% of all   MF Users had istalled by Mistake within the C:/users/downlaod folder one ( or more)  time in there life !

**Describe the solution you'd like**

Simply include a PATH field into the installer.
It should have a "hardcoded Recomendation" like  c:/mobiflight
And it should allow to rework the path..... best way with a GUI ( like Windows Explorer) .... If not just with a Path Text Line.

**Additional context**
@kkr0kk  already build this for testing.
BUT 
We got a Windows Security Warning on first try.

Whatever...: Thsi should be a point on our ToDo List !

**Comment by kkr0kk** [comment]
PR is on, i add a gui for the first install

**Comment by MobiFlight-Admin** [comment]
I will work on this later today. Thanks @kkr0kk

**Comment by pizman** [comment]
Refresh !
In case DocMoebius is not 100% happy with the current Idea...
Compromise Solution:
We not need a Custom User Path Option with a UI..... We basically just need a PROTECTION to avoid a install/unpack in the c:/users/myname/download folder.....

So maybe we could build a check like "is installer.exe currently in download folder?" if Yes not allow installation.... Give a Error Popup, and tell the user that he need to move the file.

If this is still to much as expected... Then a simple Popup before starting the install process like " Attnetion... You will install MF in the Folder where you currently start the Installer.... Do you really want to continue ??   YES/NO.

**Overall... We jsut need to avoid the user install it by fault in the Download Folder !**

**Comment by DocMoebiuz** [comment]
superseded by https://github.com/MobiFlight/MobiFlight-Connector/issues/953
